.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: Build the candy-board project
+
+on:
+  push
+  # push:
+  #   branches:
+  #   - main
+  # release:
+  #   types: [created, published]
+
+jobs:
+  build_project:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
+    - run: yarn install
+    - run: yarn build
+    - uses: actions/upload-artifact@v3
+      with:
+        name: dist-artifact
```

Comment:
Cand you specify a better Candy name ? 😸

Reply:
Done